stdlib/int64.gr

```diff
@@ -12,9 +16,32 @@ import {
   coerceInt64ToNumber as toNumber
 } from "runtime/numbers"
 
+/**
+ * @section Values: Functions for working with the `Int64` data type.
+ */
+
+/**
+ * Coverts the argument from a `Number` to a `Int64`.
+ *
+ * @param x: The `Number` to convert
+ * @returns The converted `Number`
```

Comment:
@spotandjake looks like some of the docs have backticks throughout. We don't want those (as per the Float64 docs)